mindmeld/auto_annotator.py

```diff
@@ -19,7 +19,7 @@
 from enum import Enum
 from tqdm import tqdm
 import spacy
-
+from google.cloud import translate_v2
```

Comment:
I think we can add a conditional here saying to use auto_annotator, you have to use `google.cloud` extra requirements field.